homeassistant/components/oralb/sensor.py

```diff
@@ -124,3 +140,11 @@ class OralBBluetoothSensorEntity(
     def native_value(self) -> str | int | None:
         """Return the native value."""
         return self.processor.entity_data.get(self.entity_key)
+
+    async def async_added_to_hass(self) -> None:
+        """Add subscription when added to hass."""
+        self.entity_description: OralBSensorDescription
+        if self.entity_description.requires_active_connection:
+            self.async_on_remove(
+                self.processor.coordinator.register_active(self.entity_key)
+            )
```

Comment:
I get an error if I don't type the entity description, and I get an error for the coordinator. I checked and they are both the correct types, but how can I make sure the typing is correct. Do I need to subclass processor?